example_flutter/pubspec.lock

```diff
@@ -0,0 +1,398 @@
+# Generated by pub
```

Comment:
pubspec.lock shouldn't be included.  We should at `*.lock` to our `.gitignore` file.

Reply:
I take that back.  Looks good!